src/components/bulkConcepts/addBulkConcepts.jsx

```diff
@@ -154,6 +154,11 @@ export class AddBulkConcepts extends Component {
               <h3>Concept IDs to add</h3>
             </div>
           </div>
+          <small className="form-text text-muted">
```

Comment:
Could you remove the back-ticks(`)? They might be confusing for some people.

Reply:
Done